### PR TITLE
Render context AND Log fixes

### PR DESCRIPTION
**Pull request by Mon1Amour4**
I spend and hour to make it cool (to have separate PR fo log and render context but i failed so take it easy

### CODE REVIEW

**Comment by LtSnail** [comment]
> I spend and hour to make it cool (to have separate PR fo log and render context but i failed so take it easy

Okay, no worries, it's okay to have PRs with AND occasionally, but it would be good to preserve clean approach in the long run

File: Engine/External/glad/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+project ( Glad )
+
+set( SOURCES
+    src/gl.c
+    src/wgl.c
+)
+
+set( HEADERS
+    include/glad/gl.h
+	include/glad/wgl.h
```
Comment (LtSnail) [inline Engine/External/glad/CMakeLists.txt:10]: Incorrect indent

File: Engine/Fuego/Windows/OpenGL/OpenGLContext.cpp [hunk 2]
```diff
@@ -0,0 +1,56 @@
+﻿#include "OpenGLContext.h"
+
+// clang-format off
```
[the remark this answers is not in the document]
Comment (Mon1Amour4) [reply Engine/Fuego/Windows/OpenGL/OpenGLContext.cpp:3]: cause glad must be after fupch.h

File: Engine/CMakeLists.txt [hunk 3]
```diff
@@ -128,19 +129,18 @@ target_include_directories( FuegoEngine
...
 if ( WIN32 )
     #GLAD
-    set( GLAD_SOURCE_DIR 
-        External/glad
-    )
-
-    list( APPEND 
-        SOURCES ${GLAD_SOURCE_DIR}/src/glad.c
+    target_link_libraries ( FuegoEngine 
+        PRIVATE
```
Comment (artem-zyktin) [inline Engine/CMakeLists.txt:133]: Different code style here

File: Engine/CMakeLists.txt [hunk 4]
```diff
@@ -128,19 +129,18 @@ target_include_directories( FuegoEngine
...
+        Glad
     )
 
     target_include_directories( FuegoEngine 
         PRIVATE 
-        ${GLAD_SOURCE_DIR}/include
+        External/glad/include
     )
 
+    target_link_libraries(FuegoEngine PRIVATE opengl32)
```
Comment (artem-zyktin) [inline Engine/CMakeLists.txt:142]: Different code style (see above)

File: Engine/External/glad/CMakeLists.txt (hunk 1 above)
Comment (artem-zyktin) [inline Engine/External/glad/CMakeLists.txt:10]: Bad code style

Comment (Mon1Amour4) [reply Engine/External/glad/CMakeLists.txt:10]: image

File: Engine/Fuego/Windows/WindowWin.cpp [hunk 5]
```diff
@@ -172,9 +172,15 @@ WindowWin::WindowWin(const WindowProps& props, EventQueue& eventQueue)
                             nullptr);
     FU_CORE_ASSERT(m_Hwnd, "[AppWindow] hasn't been initialized!");
     hwndMap.emplace(m_Hwnd, this);
+
+    _context = new OpenGLContext(&m_Hwnd);
+    FU_CORE_ASSERT(_context->Init(),
+                   "[GraphicsContext] hasn't been initialized!");
+
```
Comment (artem-zyktin) [inline Engine/Fuego/Windows/WindowWin.cpp:179]: That line is too short to  move to another line

Comment (Mon1Amour4) [reply Engine/Fuego/Windows/WindowWin.cpp:179]: clang-format is at work

**Review by artem-zyktin — COMMENTED** [review]
Almost good

**Review by LtSnail — APPROVED** [review]

**Pull request merged**